ZeroTouchEssentials/ZeroTouchEssentials.csproj

```diff
@@ -30,11 +33,12 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="ProtoGeometry">
-      <HintPath>..\..\Dynamo\extern\ProtoGeometry\ProtoGeometry.dll</HintPath>
+    <Reference Include="DynamoServices, Version=1.0.0.983, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>$(DynamoZeroTouchPath)\DynamoServices.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="ProtoInterface">
-      <HintPath>..\..\Dynamo\extern\ProtoGeometry\ProtoInterface.dll</HintPath>
+    <Reference Include="ProtoGeometry">
+      <HintPath>$(DynamoZeroTouchPath)\ProtoGeometry.dll</HintPath>
```

Comment:
The confusion is caused because the old ZeroTouch package contained DynamoServices. I have updated the PR according to the new Nuget packages.